wit-0.3.0-draft/timezone.wit

```diff
@@ -3,53 +3,33 @@ package wasi:clocks@0.3.0;
 @unstable(feature = clocks-timezone)
 interface timezone {
     @unstable(feature = clocks-timezone)
-    use wall-clock.{datetime};
+    use system-clock.{instant};
 
-    /// Return information needed to display the given `datetime`. This includes
-    /// the UTC offset, the time zone name, and a flag indicating whether
-    /// daylight saving time is active.
+    /// Return the IANA identifier of the currently configured timezone. The id
+    /// `UTC` indicates Coordinated Universal Time. Otherwise, this should be an
+    /// identifier from the IANA Time Zone Database.
     ///
-    /// If the timezone cannot be determined for the given `datetime`, return a
-    /// `timezone-display` for `UTC` with a `utc-offset` of 0 and no daylight
-    /// saving time.
-    @unstable(feature = clocks-timezone)
-    display: func(when: datetime) -> timezone-display;
-
-    /// The same as `display`, but only return the UTC offset.
+    /// For displaying to a user, the identifier should be converted into a
+    /// localized name by means of an internationalization API.
+    ///
+    /// In implementations that do not expose an actual time zone, this
```

Comment:
```suggestion
    /// In implementations that do not expose an actual time zone, or
    /// that are unable to provide mappings from times to deltas
    /// between the configured timezone and UTC, this
```
maybe? To make it clear that if you're returning a non-UTC value here then you must be able to support `utc-offset` for that value. Assuming `utc-offset` is not made to return an optional value, anyway.